scout/server/blueprints/cases/templates/cases/gene_panel.html

```diff
@@ -152,40 +152,58 @@ <h6 class="ms-3 mt-3"><span class="fas fa-clipboard-list"></span>&nbsp;HPO gene
             </div>
           </form>
         </div>
-        <div class="col d-flex justify-content-end">
+        <div class="row mt-3">
           <div class="btn-group btn-group-sm">
           {% if case.track == 'cancer' %}
-            <a class="btn btn-secondary btn-sm text-white"
-              href="{{ url_for('variants.cancer_variants', institute_id=institute._id,
-                               case_name=case.display_name, variant_type='clinical',
-                               gene_panels=['hpo']) }}">
-               Clinical HPO SNVs
-             </a>
-            <a class="btn btn-secondary btn-sm text-white"
-              href="{{ url_for('variants.cancer_sv_variants', institute_id=institute._id,
-                               case_name=case.display_name, variant_type='clinical',
-                               gene_panels=['hpo']) }}">
-               Clinical HPO SVs
-             </a>
+            {% if case.vcf_files.vcf_cancer %}
+              <a class="btn btn-secondary btn-sm text-white"
+                href="{{ url_for('variants.cancer_variants', institute_id=institute._id,
+                                 case_name=case.display_name, variant_type='clinical',
+                                 gene_panels=['hpo']) }}">
+                 Clinical HPO SNVs
+               </a>
+            {% endif %}
+            {% if case.vcf_files.vcf_cancer_sv %}
+              <a class="btn btn-secondary btn-sm text-white"
+                href="{{ url_for('variants.cancer_sv_variants', institute_id=institute._id,
+                                 case_name=case.display_name, variant_type='clinical',
+                                 gene_panels=['hpo']) }}">
+                 Clinical HPO SVs
+               </a>
+            {% endif %}
           {% else %}
-            <a class="btn btn-secondary btn-sm text-white"
-              href="{{ url_for('variants.variants', institute_id=institute._id,
-                               case_name=case.display_name, variant_type='clinical',
-                               gene_panels=['hpo']) }}">
-               Clinical HPO SNVs
-             </a>
-            <a class="btn btn-secondary btn-sm text-white"
-              href="{{ url_for('variants.sv_variants', institute_id=institute._id,
+            {% if case.vcf_files.vcf_snv %}
+              <a class="btn btn-secondary btn-sm text-white"
+                href="{{ url_for('variants.variants', institute_id=institute._id,
+                                 case_name=case.display_name, variant_type='clinical',
+                                 gene_panels=['hpo']) }}">
+                 Clinical HPO SNVs
+              </a>
+            {% endif %}
+            {% if case.vcf_files.vcf_sv %}
```

Comment:
Great to check: we do have an (incomplete?) set of bools on case like `case["has_strvariants"]` `case["has_svvariants"]` that are supposed to encapsulate that a little. 😊